drivers/at24cxxx/Kconfig

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+choice
+    bool "AT24CXXX EEPROM"
+    depends on HAS_PERIPH_I2C
+    depends on TEST_KCONFIG
+    optional
+    help
+        This driver supports the multiple models, select one.
+
+config MODULE_AT24C01
+    bool "AT24C01"
+    select MODULE_AT24CXXX
+
+config MODULE_AT24C32
+    bool "AT24C32"
+    select MODULE_AT24CXXX
+
+config MODULE_AT24C64
+    bool "AT24C64"
+    select MODULE_AT24CXXX
+
+config MODULE_AT24C128
+    bool "AT24C128"
+    select MODULE_AT24CXXX
+
+config MODULE_AT24C256
+    bool "AT24C256"
+    select MODULE_AT24CXXX
+
+config MODULE_AT24C512
+    bool "AT24C512"
+    select MODULE_AT24CXXX
```

Comment:
It actually supports [more than that](https://github.com/RIOT-OS/RIOT/blob/master/drivers/at24cxxx/include/at24cxxx_defines.h#L328) - and even more if those parameters are supplied by the user.

Reply:
Ah thanks, I had a hard time trying to figure which models were supported, will add them!

Reply:
I've added the modules in the list. Later we can improve this by adding one more option where the user can manually enter the parameters, but for that we need to make them configurable first.